i18n/en.json

```diff
@@ -703,6 +703,34 @@
     "id": "api.command_code.name",
     "translation": "code"
   },
+  {
```

Comment:
this should be ordered alphabetically, if not it fails on CI.

Reply:
@chetanyakan ^

Reply:
Done

Reply:
Nope, you have executed the `make i18n-extract` and you removed all the enterprise translations, because you don't have the enterprise code.

Reply:
@jespino Thanks for pointing this out. 
I got confused as the behaviour of `make i18n-extract` is not the same as webapp. (which makes sense as there's no private enterprise code for the webapp)

But maybe, in a future PR, it might make sense to create a Makefile command for community's benefit to just reorder strings and not remove enterprise ones.

Reply:
Actually, I think there is some changes proposed for the i18n-extract to preserve the enterprise strings.